threads-api/README.md

```diff
@@ -235,7 +235,7 @@ const threadsAPI = new ThreadsAPI({
 });
 ```
 
-</details>
+</>
```

Comment:
um this shouldn't be deleted

Reply:
fixed in https://github.com/junhoyeo/threads-api/commit/cf5350147420b802049817f2c6277069f1046bc1